docs/intro-react-native-components.md

```diff
@@ -0,0 +1,100 @@
+---
+id: intro-react-native-components
+title: Core Components and Native Components
```

Comment:
I think that the name of this section is a bit "redundant" - what about "Components: Native and  Core"?

Reply:
I'm worried about internationalization for that one. Explicit sets of words are easier for translators to handle. Sometimes redundant is clearer for more folks!

Reply:
Makes sense :) Ok then totally ignore my comment 👍